Casks/p/psst.rb

```diff
@@ -1,25 +1,31 @@
 cask "psst" do
-  version "0.1.0,20250525.150831"
-  sha256 :no_check
+  version "2025.06.09-f80f468"
+  sha256 "1ad6058a6814da00c2a420d101737ffa6f5eb07aec2002f0c9f9a501ebaea0f8"
 
-  url "https://nightly.link/jpochyla/psst/workflows/build/main/Psst.dmg.zip",
-      verified: "nightly.link/jpochyla/psst/workflows/build/main/"
+  url "https://github.com/jpochyla/psst/releases/download/rolling/Psst.dmg",
+      verified: "github.com/jpochyla/psst/"
   name "Psst"
-  desc "Spotify client"
+  desc "Fast and native Spotify client"
```

Comment:
```suggestion
  desc "Spotify client"
```

The previous description was more suitably following the `desc` stanza guidelines.